app/views/lists/show.html.erb

```diff
@@ -14,7 +15,8 @@
         <tr>
           <td><%= task.description %></td>
           <td><%= check_box_tag "complete", task.id %></td>
-          <td><%= link_to "Delete", task, method: :delete, data: { confirm: 'Are you sure?' } %>
+          <td><%= link_to "Edit", "#{@list.id}/tasks/#{task.id}/edit" %> </td>
```

Comment:
For some reason, `edit_list_task_path(task)` does not work here. It ends up switching the list and task id's in the URL. Any hints on why that would be happening?